index.html

```diff
@@ -11912,14 +11912,14 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p class="note"><pref>aria-relevant</pref> values of removals or all are to be used sparingly. Assistive technologies only need to be informed of content removal when its removal represents an important change, such as a buddy leaving a chat room.</p>
 				<p class="note">Text removals should only be considered relevant if one of the specified values is 'removals' or 'all'. For example, for a text change from 'foo' to 'bar' in a live region with a default <pref>aria-relevant</pref> value, the text addition ('bar') would be spoken, but the text removal ('foo') would not.</p>
 				<p><pref>aria-relevant</pref> is an optional attribute of live regions. This is a suggestion to <a>assistive technologies</a>, but assistive technologies are not required to present changes of all the relevant types.</p>
 				<p>When <pref>aria-relevant</pref> is not defined, an element's value is inherited from the nearest ancestor with a defined value. Although the value is a <a href="#valuetype_token_list">token list</a>, inherited values are not additive; the value provided on a descendant element completely overrides any inherited value from an ancestor element.</p>
-				<p>When text changes are denoted as relevant, user agents MUST monitor any descendant node change that affects the <a href="#textalternativecomputation">accessible name and description computation</a> of the live region as if the accessible name were determined from contents (<a href="#namecalculation">nameFrom: contents</a>). For example, a text change would be triggered if the HTML <code>alt</code> attribute of a contained image changed. However, no change would be triggered if there was a text change to a node outside the live region, even if that node was referenced (via <pref>aria-labelledby</pref>) by an element contained in the live region.</p>
+				<p>When text changes are denoted as relevant, user agents MUST monitor any descendant node change that affects the <a href="#textalternativecomputation">accessible name and description computation</a> of the live region as if the accessible name of the live region itself were determined from contents (<a href="#namecalculation">nameFrom: contents</a>). For example, a text change would be triggered if the HTML <code>alt</code> attribute of a contained image changed. However, no change would be triggered if there was a text change to a node outside the live region, even if that node was referenced (via <pref>aria-labelledby</pref>) by an element contained in the live region.</p>
```

Comment:
So this clarifies things a bit, but the paragraph in question still strikes me as confusing.

Regarding the sentence you're changing, could we instead say:

    When text changes are denoted as relevant, user agents MUST monitor
    all descendant nodes for changes to their content and name.

?

If you agree with the above change, I would be tempted to remove the sentence beginning with "However". After all, my proposed change says to monitor all *descendant* nodes. Pointing out that nodes which are not descendant nodes are not monitored seems superfluous.